README.md

```diff
@@ -101,24 +103,16 @@ Oh, but don't say the "F" word unless the client does first, please 😜
 
 ## What has worked for us in the past
 
-We've found the following things work really well working with clients:
+Establishing trust with clients can make the difference between getting the job done and exceeding expectations, [afraser](https://github.com/afraser) has some opinions on this stuff, but here's a high-level:
 
 - Communicate. A lot.
-- Get shit done.
-- Speak from experience.
-  - Explain how things made a team or users happy or unhappy.
-  - There is no "best practice", there's only practice.
-- Present ideas frequently.
-  - Get decision makers on your side and/or find a [wolf pack](https://www.urbandictionary.com/define.php?term=Wolfpack)
-- Build it, then show and tell.
-  - New ideas are much more likely to get pulled in if they're already "mostly done."
-- Make developers' lives easier.
-  - Automate things. Improve performance. Find pain points and fix them.
-- Transcend your team.
-  - Communicate and become best friends with Design, UX, Product Owners, Architecture, DevOps... Etc.
-- Pay attention to detail.
-  - Don’t get sloppy (slow code, bad naming, missed edge cases, etc.) .
-  - Delight developers, designers and users.
+- Deliver on promises.
+- Speak from experience (and in terms of what makes people happy or unhappy)
```

Comment:
Present ideas frequently is gone.  I would add it back and maybe even add to it:

> Present ideas frequently and get feedback.  Get decisionmakers on your side.

Reply:
I actually rolled the decisionmaker bit into "Transcend your team." ... I guess I was hesitant about this one because it didn't feel as put-together of an idea. I'll try and fit it back in.